engines/ags/detection_tables.h

```diff
@@ -8723,6 +8740,7 @@ const AGSGameDescription GAME_DESCRIPTIONS[] = {
 	GAME_ENTRY_EN("thedwarvendaggerofblitz", "teste2.exe", "06a03fe35791b0578068ab1873455463", 9874658),
 	GAME_ENTRY_EN("thedwarvendaggerofblitzch2", "TDDOB.exe", "82da2565c456dcfb265ded6fe3189c0b", 69210852),
 	GAME_ENTRY_EN("theelevator", "Elevator.exe", "f10516e88ec858700804ee69d041aead", 29506915),  // Windows
+	GAME_ENTRY_EN("theelevator", "Elevator.exe", "3b7cceb3e4bdb031dc5d8f290936e94b", 6563864),  // Windows
```

Comment:
As you can see in commit 27da99f608e33ce9683505229998aa447473e9f2, there was a mixup between "The Elevator" and "The Elevator: Rise from the Abyss". This checksum refers to the OROW version of `elevatorriseabyss`, which is already present

Reply:
Fix committed. 

Now:

- The Elevator - Rise from the Abyss! (English)
- The Elevator - Rise from the Abyss! (OROW⁄English)
- The Elevator (English)

are correctly recognized